develop/parachains messaging admo updates (#320)
polkadot-developers/polkadot-docs: develop/parachains/customize-parachain/add-existing-pallets.md

- **`version`** - the specific version of the crate to import
- **`default-features`** - determines the behavior for including pallet features when compiling the runtime with standard Rust libraries

!!! tip

>     If you’re importing a pallet that isn’t available on [`crates.io`](https://crates.io/){target=\_blank}, you can specify the pallet's location (either locally or from a remote repository) by using the `git` or `path` key. For example:

Reviewer: ```suggestion
    If you're importing a pallet that isn't available on [`crates.io`](https://crates.io/){target=\_blank}, you can specify the pallet's location (either locally or from a remote repository) by using the `git` or `path` key. For example:
```

]
```

> This section specifies the default feature set for the runtime, which includes the `std` features for each pallet. When the runtime is compiled with the `std` feature set, the standard library features for all listed pallets are enabled. If you forget to update the features section in the `Cargo.toml` file, you might encounter `cannot find function` errors when compiling the runtime.

Reviewer 2: I don't know if this is correct. Do you have the source of this error?

Author: I literally took the text out of the note without changes beyond that so if it's wrong it was wrong before me. I am not familiar with Rust or this error myself.

This basic structure shows that every pallet must define certain types, such as `RuntimeEvent` and `WeightInfo`, to function within the runtime. The actual implementation can vary depending on the pallet’s specific needs.

> ### Example - Utility Pallet

Reviewer: Not sure about the hyphen, I don't love how it looks in the TOC. Maybe "Utility Pallet Example" instead?

> ### Example - Utility Pallet

Author: ```suggestion
### Utility Pallet Example
```

> For instance, in the [`utility pallet`](https://github.com/paritytech/polkadot-sdk/tree/{{dependencies.polkadot_sdk.stable_version}}/substrate/frame/utility){target=\_blank}, the `Config` trait is implemented with the following types:

Reviewer: ```suggestion
For instance, in the [`utility`](https://github.com/paritytech/polkadot-sdk/tree/{{dependencies.polkadot_sdk.stable_version}}/substrate/frame/utility){target=\_blank} pallet, the `Config` trait is implemented with the following types:
```
